src/main/java/pl/noip/lolstats/lol/stats/utils/NoBearerException.java

```diff
@@ -0,0 +1,4 @@
+package pl.noip.lolstats.lol.stats.utils;
+
+public class NoBearerException extends Exception {
```

Comment:
InvalidBearerFormat exception would be better name

extend RuntimeException instead of Exception